source/opt/pass.h

```diff
@@ -71,6 +71,10 @@ class Pass {
     return context()->get_def_use_mgr();
   }
 
+  analysis::DebugInfoManager* get_debug_info_mgr() const {
+    return context()->get_debug_info_mgr();
+  }
+
```

Comment:
The functions like this that already exist are a relic.  I do not like getting the analysis from the pass because it gives the impression they belong to the pass.  I would prefer if this was not added.